README.md

```diff
@@ -50,10 +53,10 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
 Your Answer:
 
 ```text
-___: Defines the distance between an element's border and adjacent elements' borders.
-___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
-___: Defines the width of an element.
+margin: Defines the distance between an element's border and adjacent elements' borders.
+outline: Inserts a "wall" around an element.
+padding: Defines the distance between an element's content and its border.
+border: Defines the width of an element.
 ___: Overlays a "wall" on top of an element.
 ___: Defines the distance between the center of an element and the center of the adjacent element.
 ```
```

Comment:
Close. The correct answers are...

```
margin: Defines the distance between an element's border and adjacent elements' borders.
border: Inserts a "wall" around an element.
padding: Defines the distance between an element's content and its border.
___: Defines the width of an element.
outline: Overlays a "wall" on top of an element.
___: Defines the distance between the center of an element and the center of the adjacent element.
```

The difference between `border` and `outline` can be confused. Although to be honest, you'll rarely be using `outline` (if ever). `width` determines the width of an element.